lib/platform/azure/index.spec.ts

```diff
@@ -1,7 +1,7 @@
 import is from '@sindresorhus/is';
 import { REPOSITORY_DISABLED } from '../../constants/error-messages';
 import { BranchStatus } from '../../types';
-import * as _gitfs from '../../util/gitfs';
+import * as _git from '../../util/gitfs/git';
```

Comment:
import from `test/util` ?

Reply:
Unfortunately not so easy, these two namespaces (`platform/*` and `npm/extract`) brought me massive headaches when I tried

Reply:
So maybe we should refactor these tests to use regular mocking before this pr?

Reply:
Let's do this later, I already have a pool of refactorings which I planned to make once we get rid of `extra-fs` usage outside of `gitfs`